ui/ducks/bridge/actions.ts

```diff
@@ -27,6 +30,9 @@ const callBridgeControllerMethod = <T>(
   };
 };
 
+const isProviderConfig = (n: unknown): n is ProviderConfig =>
+  typeof n === 'object' && n !== null && 'id' in n;
+
```

Comment:
I would consider just using the chainId as a hex w/ a selector to get the config in a future PR to avoid the wrappers here